include/zephyr/irq.h

```diff
@@ -111,6 +111,51 @@ irq_connect_dynamic(unsigned int irq, unsigned int priority,
 #define IRQ_DIRECT_CONNECT(irq_p, priority_p, isr_p, flags_p) \
 	ARCH_IRQ_DIRECT_CONNECT(irq_p, priority_p, isr_p, flags_p)
 
+/**
+ * @brief Initialize a 'shared' interrupt handler.
+ *
+ * This routine initializes a shared interrupt handler for an IRQ. The IRQ
+ * must be subsequently enabled via irq_enable() before the interrupt handler
+ * begins servicing interrupts.
+ *
+ * Up to three ISRs can share one IRQ and they are called in sequence but
```

Comment:
The Kconfig file states `This option allows up to 4 ISRs to share 1 IRQ line` ... presumably this should be four (`MAX_SHARED_ISRS = 4`).